src/templates/homepage.html

```diff
@@ -0,0 +1,25 @@
+<div class="app-preview"></div>
+
+<div class="app-list">
+  {% defer (url=api('apps'), pluck='objects', as='apps', key='slug') %}
+    <ul class="clearfix">
+      {% for app in this %}
+        <li class="app-list-app focusable" data-slug="{{ app.slug }}"
+            contextmenu="contextmenu-{{ loop.index }}">
+          <img src="{{ app.icons['128'] }}"
+               alt="{{ app.name }}" title="{{ app.name }}">
+          <span class="name">{{ app.name }}</span>
+
+          <menu type="context" id="contextmenu-{{ loop.index }}">
```

Comment:
Yes, rest of the work will be done by the system app. Its in this [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1225086). By the way, do we need to disable the context menu if the marketplace app is not running on the TV?